packages/bpk-foundations-ios/src/base/typography.json

```diff
@@ -91,128 +51,68 @@
-    },
-    "TEXT_CAPS_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_BOOK}",
-      "type": "string",
-      "category": "font-weights"
-    },
     "TEXT_XS_LETTER_SPACING": {
       "value": "{!LETTER_SPACING_XS}",
       "type": "legacy-letter-spacing",
       "category": "letter-spacings"
     },
-    "TEXT_XS_FONT_SIZE": {
-      "value": "{!FONT_SIZE_XS}",
-      "type": "font-size",
-      "category": "font-sizes"
-    },
-    "TEXT_XS_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_BOOK}",
-      "type": "string",
-      "category": "font-weights"
-    },
     "TEXT_SM_LETTER_SPACING": {
       "value": "{!LETTER_SPACING_SM}",
       "type": "legacy-letter-spacing",
       "category": "letter-spacings"
     },
-    "TEXT_SM_FONT_SIZE": {
-      "value": "{!FONT_SIZE_SM}",
-      "type": "font-size",
-      "category": "font-sizes"
-    },
-    "TEXT_SM_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_BOOK}",
-      "type": "string",
-      "category": "font-weights"
-    },
     "TEXT_BASE_LETTER_SPACING": {
       "value": "{!LETTER_SPACING_BASE}",
       "type": "legacy-letter-spacing",
       "category": "letter-spacings"
     },
-    "TEXT_BASE_FONT_SIZE": {
-      "value": "{!FONT_SIZE_BASE}",
-      "type": "font-size",
-      "category": "font-sizes"
-    },
-    "TEXT_BASE_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_BOOK}",
-      "type": "string",
-      "category": "font-weights"
-    },
     "TEXT_LG_LETTER_SPACING": {
       "value": "{!LETTER_SPACING_LG}",
       "type": "legacy-letter-spacing",
       "category": "letter-spacings"
     },
-    "TEXT_LG_FONT_SIZE": {
-      "value": "{!FONT_SIZE_LG}",
-      "type": "font-size",
-      "category": "font-sizes"
-    },
-    "TEXT_LG_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_BOOK}",
-      "type": "string",
-      "category": "font-weights"
-    },
     "TEXT_XL_LETTER_SPACING": {
       "value": "{!LETTER_SPACING_XL}",
       "type": "legacy-letter-spacing",
       "category": "letter-spacings"
     },
-    "TEXT_XL_FONT_SIZE": {
-      "value": "{!FONT_SIZE_XL}",
-      "type": "font-size",
-      "category": "font-sizes"
-    },
-    "TEXT_XL_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_BOOK}",
-      "type": "string",
-      "category": "font-weights"
-    },
     "TEXT_XXL_LETTER_SPACING": {
       "value": "{!LETTER_SPACING_XXL}",
       "type": "legacy-letter-spacing",
       "category": "letter-spacings"
     },
-    "TEXT_XXL_FONT_SIZE": {
-      "value": "{!FONT_SIZE_XXL}",
-      "type": "font-size",
-      "category": "font-sizes"
-    },
-    "TEXT_XXL_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_BOOK}",
-      "type": "string",
-      "category": "font-weights"
-    },
     "TEXT_XXXL_LETTER_SPACING": {
       "value": "{!LETTER_SPACING_XXXL}",
       "type": "legacy-letter-spacing",
       "category": "letter-spacings"
     },
-    "TEXT_XXXL_FONT_SIZE": {
-      "value": "{!FONT_SIZE_XXXL}",
+    "TEXT_XL_HEAVY_FONT_SIZE": {
```

Comment:
how come the heavy ones are only on iOS? we'll need them for legacy on android too

Reply:
ah great! yeah, they weren't defined on android I guess. I'll put em on commons!